plugins/callback/foreman.py

```diff
@@ -20,16 +20,20 @@
       - requests (python library)
     options:
       url:
-        description: URL to the Foreman server
+        description:
+          - URL of the Foreman server.
         env:
           - name: FOREMAN_URL
+          - name: FOREMAN_SERVER_URL
+          - name: FOREMAN_SERVER
```

Comment:
This looks like we'd want all three combinations in the other places too, right?

Reply:
preferably not? but I can be convinced ;)

Reply:
:cyclone: _convince_ :cyclone:

Reply:
very convincing tornado

Reply:
for consistency!